libs/platform/kernel/ebpf_platform_kernel.c

```diff
@@ -212,7 +212,7 @@ ebpf_allocate_ring_buffer_memory(size_t length)
     }
 
 #pragma warning(push)
-#pragma warning(disable : 28145) /* The opaque MDL structure should not be modified by a driver except for
+#pragma warning(disable : 28145) /* The opaque MDL structure should not be modified by a driver except for \
```

Comment:
Why is this change needed?

Reply:
I asked this on Alan's PR and he said the code formatter added it automatically.